pom.xml

```diff
@@ -1,161 +1,364 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!--
+
+    Copyright 2016-2017 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud.trace.adapters.zipkin</groupId>
-  <artifactId>parent</artifactId>
-  <name>Stackdriver Zipkin Adapters Parent Project</name>
-  <description>
-    Adapters for using Zipkin instrumentation with Stackdriver Trace.
-  </description>
-  <url>http://cloud.google.com/trace</url>
-  <version>0.4.0-SNAPSHOT</version>
+  <groupId>io.zipkin.gcp</groupId>
+  <artifactId>zipkin-gcp-parent</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
+  <modules>
+    <module>autoconfigure</module>
+    <module>storage-stackdriver</module>
+  </modules>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
+
+    <!-- default bytecode version for src/main -->
+    <main.java.version>1.6</main.java.version>
+    <main.signature.artifact>java16</main.signature.artifact>
+
+    <main.basedir>${project.basedir}</main.basedir>
+
+    <license-maven-plugin.version>2.11</license-maven-plugin.version>
+
+    <slf4j.version>1.7.25</slf4j.version>
+    <!-- only used for protos, we could possibly obviate this if a problem -->
+    <grpc-google-devtools-cloudtrace-v1.version>0.1.1</grpc-google-devtools-cloudtrace-v1.version>
+
+    <!-- must match (ex. use spring-boot used by zipkin-server) -->
+    <zipkin-reporter2.version>2.1.4</zipkin-reporter2.version>
+    <zipkin.version>2.2.3-SNAPSHOT</zipkin.version>
```

Comment:
awaiting a netty-free zipkin release (as we are overriding netty and boringssl to match grpc)